xen-bugtool

```diff
@@ -213,6 +213,8 @@ XAPI_CLUSTERD = '/var/opt/xapi-clusterd/db'
 SWTPM_IC = '/var/lib/swtpm-localca/issuercert.pem'
 SWTPM_RCA ='/var/lib/swtpm-localca/swtpm-localca-rootca-cert.pem'
 SWTPM_CS = '/var/lib/swtpm-localca/certserial'
+NRPE_CONF = '/etc/nagios/nrpe.cfg'
+NRPE_CUSTOMIZABLE_CONF = '/etc/nrpe.d'
```

Comment:
Is this a file name or a directory name? 
it looks like a directory name, and if yes, `tree_output` should be used for this.